ui/components/multichain/multichain-components.scss

```diff
@@ -0,0 +1,9 @@
+/**
+* Please import your styles in order of atomicity.
+* The most atomic styles should be imported first.
+* This will help improve specificity and reduce the chance of
+* unintended overrides.
+**/
+@import 'account-list-item/index';
+@import 'account-list-menu/index';
+@import 'multichain-token-list-item/multichain-token-list-item';
```

Comment:
@darkwing I have removed index.scss and replaced it with `multichain-components.scss` to keep in sync with how we have named the default global scss files in component-library, app and ui folder.